src/lib/fetchApi.js

```diff
@@ -1,6 +1,7 @@
 import 'whatwg-fetch';
 
-const fetchApi = endpoint =>
-  fetch(`/api/${endpoint}`).then(response => response.json());
+const fetchApi = endpoint => fetch(`/api/${endpoint}`).then(response => response.json());
+
+export const fetchUrl = url => fetch(url).then(response => response.json());
```

Comment:
not a blocker for now but I think having two methods just to not retype "api" is a bit exaggerated/over-engineered... I suggest either simply removing "api" from the server endpoint url .. or merge the two methods and include "api" in the url manually.. (I prefer the 1st solution :D )   @milenvoutchev